packages/date/src/index.js

```diff
@@ -379,6 +380,17 @@ export function isInTheFuture( dateValue ) {
 	return momentObject.isAfter( now );
 }
 
+/**
+ * Get the moment object from date.
+ *
+ * @param {string} dateValue Date String or Date object in the Defined WP Timezone.
+ *
+ * @return {Moment} The Moment object.
+ */
+export function getMomemt( dateValue ) {
```

Comment:
There's a typo here, I'm guessing it should be `getMoment`.

Reply:
Also, just wanted to check one other thing. Is the intended use of this function only as a helper in the test?

Reply:
> There's a typo here, I'm guessing it should be `getMoment`.

Oh yes! It should be `getMoment`, will fix that.

> Is the intended use of this function only as a helper in the test?

No, the intention is to expose moment API for other uses.